meerk40t/extra/vectrace.py

```diff
@@ -1,40 +1,19 @@
-from meerk40t.svgelements import Matrix, Path, Polygon
+from meerk40t.svgelements import Matrix, Path, Polygon, Color
 
 
 def plugin(kernel, lifecycle=None):
     if lifecycle == "register":
         _ = kernel.translation
 
-        @kernel.console_command(
-            "vectrace",
-            help=_("return paths around image"),
-            input_type="image",
-            output_type="elements",
-        )
-        def vectrace(data, **kwargs):
-            elements = kernel.root.elements
-            path = Path(fill="black", stroke="blue")
-            paths = []
-            for node in data:
-                matrix = node.matrix
-                image = node.image
-                width, height = node.image.size
-                if image.mode != "L":
-                    image = image.convert("L")
-                image = image.point(lambda e: int(e > 127) * 255)
-                for points in _vectrace(image.load(), width, height):
-                    path += Polygon(*points)
-                path.transform *= Matrix(matrix)
-                paths.append(
-                    elements.elem_branch.add(
-                        path=abs(path),
-                        stroke_width=0,
-                        stroke_scaled=False,
-                        type="elem path",
-                    )
-                )
-            return "elements", paths
+        setup_vectrace(kernel)
 
+        # A very simple hack....
+        try:
+            import potrace
+            setup_potrace(kernel)
+            print("Good news - potrace found...")
+        except ModuleNotFoundError:
```

Comment:
This probably should be in it's own file completely and loaded independently and if it can't do the import, return False during the plugin initialization routines.